### DISCUSSION TITLE
Order of Achievement Badges?

### DISCUSSION THREAD

**Discussion by CPang23** [community/community#19338] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there any way to arrange the order of achievement badges? I'm guessing its based on the level of the badges, but I don't have a way of verifying this.

**Comment by melvincwng** [comment]
Hi @CPang23, unfortunately I don't think there is a way to arrange the order of your achievement badges.

Hopefully this clarifies, thanks!

If this reply has helped, please 'Mark as answer'. Thank you :)

**Comment by melvincwng** [comment, marked as answer]
@CPang23, based on my understanding, achievement badges are displayed based on the order of unlocking them.

Achievements that are unlocked earlier, will be displayed at the further end/right end of your profile page. While more recent achievements are shown at the left end of your profile page.

You can use this mechanism to artificially "order" your achievements, I guess. But other than that, there isn't any known way to manually reorder them after unlocking them.

**Reply by CPang23** [reply]
Thanks 👍
